src/app/action-items/action-items.component.ts

```diff
@@ -57,7 +58,36 @@ export class ActionItemsComponent implements OnInit {
       .subscribe(item => {
         if (!!item) {
           this.retrieveActionItems();
+          const justAddedItemId = parseInt(this.justAddedItem.id) + 1;
```

Comment:
**FIX:** This should be taken from the result of the dialog, so `item.id`. Doing this autoincrement won't guarantee you that this will be the id of the item generated by the server (it will only work for this case where you have dummy data generated in the service with autoincrement).